lib/concurrent/utility/processor_counter.rb

```diff
@@ -76,6 +78,8 @@ def physical_processor_count
       def compute_processor_count
         if Concurrent.on_jruby?
           java.lang.Runtime.getRuntime.availableProcessors
+        elsif defined?(Etc) && Etc.respond_to?(:nprocessors) && (nprocessor = Etc.nprocessors rescue nil)
```

Comment:
Etc is not included by default, so should this probe also include a `require "etc"`?

Reply:
👍 Good catch. I added it as a top-level require (I believe all supported rubies should include `etc`, older ones just don't implement `nprocessors`).